conda/recipes/cuml/build.sh

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env bash
+# Copyright (c) 2018-2022, NVIDIA CORPORATION.
 
 # This assumes the script is executed from the root of the repo directory
-./build.sh cuml
+./build.sh cuml -v
```

Comment:
Is this a debugging change?

Reply:
Not really, I much prefer to make cmake extra verbose here for diagnosing when there are CI problems